### PR TITLE
Add Brand identity guidelines

### LINKED ISSUE #2816: Add osu! identity guidelines
https://github.com/ppy/osu-web/issues/4996#issuecomment-534363999

### PR DESCRIPTION
**Pull request by MegaApplePi**
resolves #2816 

- [As noted by flyte](https://github.com/ppy/osu-web/issues/4996#issuecomment-554184420), only the osu! cookie logo guidelines were still valid.
- Also adds a link from the Glossary#cookie to the new article.

## Things to do

- [x] Add link to the design kit

### CODE REVIEW

**Comment by MegaApplePi** [comment]
I put the Identity Guidelines in the Legal folder... not sure if it shouldn't be there

**Comment by cl8n** [comment]
def not legal, make it top level

**Comment by cl8n** [comment]
and maybe "Brand identity guidelines" to make it more clear for ppl who arent familiar with this language

**Comment by MegaApplePi** [comment]
We should probably add a link to the Design Kit. Is the google drive link okay to use?

**Comment by cl8n** [comment]
id imagine it's going to end up on on assets.ppy.sh at some point, @Ephemeralis can u see about uploading this https://drive.google.com/file/d/1TmUot5nu49p71icz4u3G68njLAQOeQrG/view ? it's version 1.1 of identity guidelines

**Comment by MegaApplePi** [comment]
The pdf in that google drive link should be updated to the one that flyte linked in https://github.com/ppy/osu-web/issues/4996#issuecomment-534363999 and section b (pages 7 to 11) should be removed from that.

Unless you guys think it's fine as is.

**Comment by cl8n** [comment]
I was thinking more like 1 image for every category of logo problem, and then explain all of them in a list or something instead of title text

it's fine as-is if you want to give each individual problem this much focus, but I'm not sure if that's necessary and like said earlier this article is super spread out for relatively little info

**Comment by MegaApplePi** [comment]
Okay, I'll work on that. It took me too long to figure out what you actually meant.

**Comment by cl8n** [comment]
png might be a better format since there isn't much variation in the colors, worth a try at least
looks good though. for the design kit link ig anything hosted on google drive or mega etc is fine for now, if there's no official upload

File: wiki/Brand_identity_guidelines/en.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Brand identity guidelines
+
```
Comment (sr229) [inline wiki/Brand_identity_guidelines/en.md]: I suggest adding a opening statement here so people can understand what's the purpose of this Press Kit.

File: wiki/Brand_identity_guidelines/en.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+The brand identity guidelines is a set of standards when branding osu! related projects. All use does not require approval.
```
Comment (cl8n) [inline wiki/Brand_identity_guidelines/en.md]:
```suggestion
The brand identity guidelines are a set of standards for branding osu!-related projects. All use does not require approval.
```

File: wiki/Brand_identity_guidelines/en.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+[Download the design kit here.](https://drive.google.com/file/d/1TmUot5nu49p71icz4u3G68njLAQOeQrG/view?usp=sharing) The linked design kit contains other outdated assets; the assets that are current is the osu! cookie. Also note that the included identity guidelines PDF is outdated, please refer to this page for the brand identity guidelines.
```
Comment (cl8n) [inline wiki/Brand_identity_guidelines/en.md]:
```suggestion
[Download the design kit here.](https://drive.google.com/file/d/1TmUot5nu49p71icz4u3G68njLAQOeQrG/view?usp=sharing) The linked design kit contains some outdated assets. Also note that the included PDF is outdated and this page should be used instead.
```
removing bulky wording and I think calling everything except the osu cookie outdated isn't true.

**Pull request merged**